crates/nu-explore/src/nu_common/command.rs

```diff
@@ -90,8 +90,8 @@ fn eval_source2(
     //
     // So we LITERALLY ignore all expressions except the LAST.
     if block.len() > 1 {
-        let range = ..block.pipelines.len() - 1;
-        Arc::make_mut(&mut block).pipelines.drain(range);
+        let block = Arc::make_mut(&mut block);
+        block.pipelines.drain(..block.pipelines.len() - 1);
```

Comment:
Mhh not sure if this really makes it clearer that the `block: &mut Arc<Block>` in the outside scope now points to that mutated `Arc` inner. Which is relevant for the `eval_block` at the end of the function.

Reply:
I went back to the original order and added a comment.